internal/runbits/activation/activation.go

```diff
@@ -40,7 +40,8 @@ func ActivateAndWait(
 		}
 	}
 
-	ve, err := venv.GetEnv(false, true, projectDir)
+	bashifyPaths := rt.GOOS == "windows" && ss.Shell() != "cmd"
```

Comment:
Can we put `ss.Shell() != "cmd"` into subshell itself, and expose a method instead? eg. `ss.BashifyPaths()`.